openapi-specs/schema.json

```diff
@@ -3055,6 +3055,48 @@
             }
           }
         }
+      },
+      "delete": {
```

Comment:
one concern i have is that an admin user may be submitting these operations via curl or python and inadvertently (eg from their shell history) send a delete command instead of a post command since the url paths are the same. perhaps i'm being overly cautious. part of me thinks we should append `/all` to the url path or perhaps require the user does some sanity check like include a `delete_all=true` url parameter. i don't think this pr should be blocked by this, but if you think this is worth discussing, let's consider adding something like this in a follow up pr. i suppose we could make the same argument for `client.collections.delete_all()` and that is definitely much more dangerous, so i'm probably being overly cautious

Reply:
You raise a good point! I think adding some query parameters in this regard might help, perhaps something like `dry_run=True` or `are_you_sure=True` 😂 But to achieve a RESTful interface for this resource (`/replication/replicate`) then it makes most semantic sense to have `DELETE /replication/replicate` be deleting all operations since we have `DELETE /replication/replicate/{id}` for deleting one operation